### PR TITLE
Update TACO files

### PR DESCRIPTION
**Pull request by xiazcy**
### Summary

Update TACO files to add
 - Localization Parameters
 - Added Extract Only CAP_EXTRACT_ONLY - https://tableau.github.io/connector-plugin-sdk/docs/capabilities.html

### Description
Updating TACO files for Tableau publication

### CODE REVIEW

File: tableau-connector/src/resources-en_US.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <string name="additional_conn_options">Additional Connection Options</string>
+
+    <string name="enable_ssh_tunnel">Enable SSH Tunnel</string>
+    <string name="ssh_user">SSH User</string>
+    <string name="ssh_hostname">SSH Hostname</string>
+    <string name="ssh_private_key_file">SSH Private Key File</string>
+    <string name="ssh_private_key_file_passphrase">SSH Private Key File Passphrase</string>
+    <string name="ssh_strict_host_key_check">SSH Strict Host Key Check (disabling option is less secure)</string>
+    <string name="ssh_known_hosts_file">SSH Known Hosts File</string>
+</resources>
```
Comment (lyndonbauto) [inline tableau-connector/src/resources-en_US.xml]: Nit, missing new line.

File: tableau-connector/src/connection-fields.xml [hunk 2]
```diff
@@ -2,69 +2,72 @@
...
 <connection-fields>
 
-  <field name="server" label="Neptune Endpoint" category="endpoint" value-type="string">
+  <field name="server" label="@string/server_endpoint/" category="endpoint" value-type="string">
     <validation-rule reg-exp="^(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])$"/>
   </field>
 
-  <field name="port" label="Port" category="endpoint" value-type="string" default-value="8182"/>
+  <field name="port" label="@string/port/" category="endpoint" value-type="string" default-value="8182">
+    <!-- Validates that port range is from 0 to 65535. -->
```
Comment (lyndonbauto) [inline tableau-connector/src/connection-fields.xml]: Nit port from 1 to 65535.
```suggestion
    <!-- Validates that port range is from 1 to 65535. -->
```

File: tableau-connector/src/resources-en_US.xml (hunk 1 above)
Comment (xiazcy) [inline tableau-connector/src/resources-en_US.xml]:
```suggestion
</resources>

```

Comment (xiazcy) [reply tableau-connector/src/resources-en_US.xml]: added =D

**Review by alexey-temnikov — APPROVED** [review]

File: tableau-connector/src/connection-fields.xml [hunk 3]
```diff
@@ -2,69 +2,72 @@
...
 
-  <field name="server" label="Neptune Endpoint" category="endpoint" value-type="string">
+  <field name="server" label="@string/server_endpoint/" category="endpoint" value-type="string">
     <validation-rule reg-exp="^(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])$"/>
   </field>
 
-  <field name="port" label="Port" category="endpoint" value-type="string" default-value="8182"/>
+  <field name="port" label="@string/port/" category="endpoint" value-type="string" default-value="8182">
+    <!-- Validates that port range is from 1 to 65535. -->
+    <validation-rule reg-exp="^([0-9]{1,4}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$"/>
```
Comment (vkagamlyk) [inline tableau-connector/src/connection-fields.xml]: Regex does not filter out port 0

File: tableau-connector/src/connection-fields.xml [hunk 4]
```diff
@@ -2,69 +2,72 @@
...
 <connection-fields>
 
-  <field name="server" label="Neptune Endpoint" category="endpoint" value-type="string">
+  <field name="server" label="@string/server_endpoint/" category="endpoint" value-type="string">
     <validation-rule reg-exp="^(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])$"/>
   </field>
 
-  <field name="port" label="Port" category="endpoint" value-type="string" default-value="8182"/>
+  <field name="port" label="@string/port/" category="endpoint" value-type="string" default-value="8182">
+    <!-- Validates that port range is from 1 to 65535. -->
```
Comment (xiazcy) [inline tableau-connector/src/connection-fields.xml:10]:
```suggestion
    <!-- Validates that port range is from 0 to 65535. -->
```

File: tableau-connector/src/connection-fields.xml (hunk 3 above)
Comment (xiazcy) [reply tableau-connector/src/connection-fields.xml]: I'm less inclined to change the regex, as the current one was used by another connector. 
But the current regex doesn't filter out leading 0's either, so here is the proposed regex change (tested on regex101). 
```suggestion
    <validation-rule reg-exp="^([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$"/>
```

Comment (vkagamlyk) [reply tableau-connector/src/connection-fields.xml]: looks good

Comment (xiazcy) [reply tableau-connector/src/connection-fields.xml]: Thanks!

**Pull request merged**